src/modules/universe/RadixNode.ts

```diff
@@ -1,49 +1,28 @@
-import { RadixBytes } from '../atommodel';
+import { RadixNodeInfo } from '../..';
+import Long from 'long';
 
-export default interface RadixNode {
-    attempts?: number
-    hash?: string
-    host: {
-        ip: string
-        port: number
+export default class RadixNode {
+    constructor(readonly nodeInfo: RadixNodeInfo, readonly wsAddress: string, readonly httpAddress: string) {
+        //
```

Comment:
Commenting an empty line? What kind of witchcraft is this?

Reply:
Otherwise tslint complains about an empty code block. But you're right, it's cleaner to just move the closing bracket up